### PR TITLE
Support multi-root workspace

### PR DESCRIPTION
**Pull request by rrag**
1. Upgrade to vscode 1.1.10 to get the `getWorkspaceFolder` api
2. On change of tab refresh the files cache if workspace folder changes
3. Show info message when using plugin when workspacePath is null

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -51,11 +54,25 @@ class RelativePath {
         // Watch for file system changes - as we're caching the searched files
         this._watcher = workspace.createFileSystemWatcher("**/*.*", false, true, false);
 
+        // this._watcher.onDidChange()
```
Comment (jakob101) [inline src/extension.ts]: Please remove this

File: src/extension.ts [hunk 2]
```diff
@@ -51,11 +54,25 @@ class RelativePath {
...
         // Add a file on creation
         this._watcher.onDidCreate((e: Uri) => {
             this._items.push(e.fsPath.replace(/\\/g, "/"));
         });
 
+        // on change active text editor refresh the cache
+        // if the workspace folder has changed
+        window.onDidChangeActiveTextEditor((e: TextEditor) => {
```
Comment (jakob101) [inline src/extension.ts]: How often does this happen? On every open/view of a file?

Comment (jakob101) [reply src/extension.ts]: Doesn't matter, I see.

**Comment by jakob101** [comment]
Looks okay to me, please remove the comment and we're good to go! 
image

**Comment by iambrandonn** [comment]
@rrag are you still interested in finishing this?

**Comment by rrag** [comment]
Sorry for the delay, I forgot about this. I am pushing up the change in a min

**Comment by rrag** [comment]
Done. @jakob101 sorry for the delay again. Please review and merge

**Comment by jakob101** [comment]
No worries. Thanks! 👍 👍

**Comment by iambrandonn** [comment]
Can this be published as a new version, or is it still being tested?

**Comment by jakob101** [comment]
I’ll publish after verification on Sunday, guys :)

________________________________

**Comment by jakob101** [comment]
Done! Thanks folks!

**Pull request merged**